omero/training-server/playbook.yml

```diff
@@ -114,6 +114,12 @@
         dest: /etc/nginx/conf.d/websockets.conf
       # Don't notify, nginx isn't installed yet
 
+    - name: NGINX - omero-ms-zarr support
+      become: yes
+      template:
+        src: ../../templates/nginx-confdnestedincludes-omero-ms-zarr.j2
+        dest: /etc/nginx/conf.d-nested-includes/omero-ms-zarr.conf
+      # Don't notify, nginx isn't installed yet
```

Comment:
As you probably guessed from the comment if this is run on an existing server any change to this file will require a manual restart of `nginx`.

If you move this to the `tasks:` section you should be able to notify the nginx restart handler. Alternatively you could append it to
https://github.com/ome/prod-playbooks/blob/eb60dbda1082b02e18aafafbfaabae94761293eb/nightshade-webclients.yml#L107-L108

Arguably the preceding `NGINX - websocket proxy support` config should go in there too but that's a separate issue